### DISCUSSION TITLE
deployment error

### DISCUSSION THREAD

**Discussion by denniztr** [vercel/next.js#62093] [Help] [answered]
### Summary

Hi there!
I keep getting this issue during deployment. Can anyone suggest how to solve it?

![error](https://github.com/vercel/next.js/assets/128366775/ed318fd8-7f25-4e7e-b69b-9a22af9ed737)

### Additional information

Windows 11, 22H2
64-bit operating system, x64-based processor

### Example

https://github.com/denniztr/chat-messenger-react-js

**Comment by icyJoseph** [comment, marked as answer]
Hi,

This is a problem with bcrypt... and it is a bit annoying, basically, the linux instance that's running your code is missing some libraries :/ 

- https://github.com/paramiko/paramiko/issues/2108
- https://stackoverflow.com/questions/76345524/vercel-deployment-errors-out-when-installing-bcrypt

I've seen people having this isue end up fixing it though, but they never say how...

In your case I see that you have `node` as a dependency, try to remove it as suggested here:

- https://github.com/orgs/vercel/discussions/2867#discussioncomment-6190002

**Reply by denniztr** [reply]
Hi @icyJoseph! Removing the `node` dependency has been a solution! Thank you!
